locust/runners.py

```diff
@@ -486,6 +494,9 @@ def _start(self, user_count: int, spawn_rate: float, wait: bool = False) -> None
         if wait and user_count - self.user_count > spawn_rate:
             raise ValueError("wait is True but the amount of users to add is greater than the spawn rate")
 
+        # if user_classes is None:
+        #    user_classes = self.user_classes
+
         for user_class in self.user_classes:
             if self.environment.host:
```

Comment:
What happens if we combine a -H/--host parameter with shape-configured Users? I guess that will still work?

Reply:
So normally it should.  I'll try it out tomorrow :)

Reply:
@cyberw So I tried it with a few example locust files, and it seemed to work.

If you want to be sure, you can also try it out. :)